### PR TITLE
README.rst: Update README

### LINKED ISSUE #6092: Include Gitter chat button in README.
The current README contains a text hyperlink that links to the Gitter chat room. This could be replaced with a "Chat on Gitter" button. This could be considered a difficulty/newcomer issue.

### PR DESCRIPTION
**Pull request by 2knal**
This fixes the issue by replacing <em>chat hyperlink</em>, with ```chat on gitter badge```.

Fixes https://github.com/coala/coala/issues/6092

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `corobo mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by 2knal** [comment]
Hi there,
are the recent changes fine?

File: README.rst [hunk 1]
```diff
@@ -17,7 +17,11 @@
 coala supports popular programming languages including **Python**, **C/C++**, **Java**, **JavaScript**, **CSS**, and `several others <http://coala.io/#!/languages>`_ out of the box.
 
 
-`Install coala <http://coala.io/#!/home#installation>`_ | `Get Involved <http://coala.io/#/getinvolved>`_ | `Chat <https://coala.io/chat>`_ | `Roadmap <https://projects.coala.io>`_ | `Blog <http://blog.coala.io>`_ | `Twitter <https://www.twitter.com/coala_io>`_
+<<<<<<< HEAD
```
Comment (frextrite) [inline README.rst]: Awesome work @2knal. However, it looks like a merge-conflict resolution was interrupted in between. Please fix this by choosing the correct block. Amend the commit (`git commit --amend`) and force push the changes!

Comment (2knal) [reply README.rst]: Sorry for the mistake,
Is it fine now?

**Comment by frextrite** [comment]
@2knal Just one more thing, I think the commit body exceeds the limit of 72. Have a look at [commit message guidelines](http://api.coala.io/en/latest/Developers/Writing_Good_Commits.html)

**Comment by 2knal** [comment]
@frextrite, I've made the necessary changes but the builds seem to be failing!

**Comment by abhishalya** [comment]
Hey, thanks for this and sorry for not being able to merge this. I feel this change is not really that important for us and can be ignored for now. Closing this for now.

Also it's always recommended to **_not_** create PRs from master branch.

**Pull request closed without merging**